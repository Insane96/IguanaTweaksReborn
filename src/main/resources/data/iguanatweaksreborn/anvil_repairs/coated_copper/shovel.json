{
	"item_to_repair": {
		"id": "iguanatweaksreborn:coated_copper_shovel"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:obsidian"
			},
			"amount": 1
		}
	]
}