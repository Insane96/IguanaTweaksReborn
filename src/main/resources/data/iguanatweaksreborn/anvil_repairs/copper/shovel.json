{
	"item_to_repair": {
		"id": "iguanatweaksreborn:copper_shovel"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:copper_ingot"
			},
			"amount": 1
		}
	]
}