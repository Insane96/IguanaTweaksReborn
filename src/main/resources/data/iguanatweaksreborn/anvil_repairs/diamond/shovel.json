{
	"item_to_repair": {
		"id": "minecraft:diamond_shovel"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:diamond"
			},
			"amount": 1
		}
	]
}