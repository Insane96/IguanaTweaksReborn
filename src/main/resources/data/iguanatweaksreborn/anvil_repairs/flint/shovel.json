{
	"item_to_repair": {
		"id": "iguanatweaksreborn:flint_shovel"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:flint"
			},
			"amount": 1
		}
	]
}