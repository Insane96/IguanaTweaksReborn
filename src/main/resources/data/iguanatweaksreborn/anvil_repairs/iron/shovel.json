{
	"item_to_repair": {
		"id": "minecraft:iron_shovel"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:iron_ingot"
			},
			"amount": 1
		},
		{
			"repair_material": {
				"id": "minecraft:iron_nugget"
			},
			"amount": 9
		}
	]
}