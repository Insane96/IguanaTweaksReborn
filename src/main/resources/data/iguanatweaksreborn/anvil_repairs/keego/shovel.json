{
	"item_to_repair": {
		"id": "iguanatweaksreborn:keego_shovel"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:diamond"
			},
			"amount": 1,
			"max_repair": 0.5
		},
		{
			"repair_material": {
				"id": "iguanatweaksreborn:keego"
			},
			"amount": 2
		}
	]
}