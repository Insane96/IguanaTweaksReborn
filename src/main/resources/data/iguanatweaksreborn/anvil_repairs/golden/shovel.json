{
	"item_to_repair": {
		"id": "minecraft:golden_shovel"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:gold_ingot"
			},
			"amount": 1
		},
		{
			"repair_material": {
				"id": "minecraft:gold_nugget"
			},
			"amount": 7,
			"cost_multiplier": 9
		}
	]
}